File: python/modelo/medico.py
from datetime import date
from modelo.personal_salud import PersonalSalud
class Medico(PersonalSalud):
    def __init__(self, dni: str, nombres: str, apellidos: str, fecha_nac: date, sexo: str,
                 colegiatura: str, rne: str = ""):
        super().__init__(dni, nombres, apellidos, fecha_nac, sexo, colegiatura, "MEDICINA_GENERAL", "MAÑANA")
        self.__rne = rne
    def firmar_atencion(self, id_atencion: str) -> str:
        return f"FIRMA_CMP_{self._codigo_colegiatura}_ATENCION_{id_atencion}"
    def obtener_rol(self) -> str:
        return "MEDICO"

File: java/modelo/Medico.java
package modelo;
import java.time.LocalDate;
public class Medico extends PersonalSalud {
    private String rne;
    public Medico(String dni, String nombres, String apellidos, LocalDate fechaNacimiento,
                  String sexo, String codigoColegiatura, String rne) {
        super(dni, nombres, apellidos, fechaNacimiento, sexo, codigoColegiatura, "MEDICINA_GENERAL", "MAÑANA");
        this.rne = rne;
    }
    @Override
    public String firmarAtencion(String idAtencion) {
        return "FIRMA_CMP_" + this.codigoColegiatura + "_ATENCION_" + idAtencion;
    }
    @Override
    public String obtenerRol() {
        return "MEDICO";
    }
    public String getRne() {
        return rne;
    }
}